invenio_users_resources/services/users/service.py

```diff
@@ -36,18 +46,18 @@ def user_cls(self):
 
     @unit_of_work()
     def create(self, identity, data, raise_errors=True, uow=None):
-        """Create a user."""
+        """Create a user from users admin."""
         self.require_permission(identity, "create")
-
-        # validate data
+        # Remove None values to avoid validation issues
+        data = {k: v for k, v in data.items() if v}
```

Comment:
drops False/empty strings: should this be `v is not None` to avoid hiding validation?